core/src/main/java/org/jdbi/v3/core/Handle.java

```diff
@@ -78,8 +77,8 @@ void setConfig(ConfigRegistry config) {
         this.config.set(config);
     }
 
-    void setConfigThreadLocal(ThreadLocal<ConfigRegistry> config) {
-        this.config = config;
+    void setConfigThreadLocal(ThreadLocal<ConfigRegistry> conf) {
+        this.config = conf;
```

Comment:
When you make changes like this, `this.` becomes redundant.  Should we clean those up as well if we forbid name shadowing?